docs/tutorials/intro_tutorial.ipynb

```diff
@@ -1401,6 +1274,131 @@
     "print(results_one_episode.to_string(index=False, columns=[\"Step\", \"Gini\"], max_rows=25))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Analyzing model reporters: Comparing 5 scenarios\n",
+    "Other insight might be gathered when we compare the Gini coefficient of different scenarios. For example, we can compare the Gini coefficient of a population with 25 agents to the Gini coefficient of a population with 400 agents. While doing this, we increase the number of iterations to 25 to get a better estimate of the Gini coefficient for each population size and get usable error estimations."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [
+    "params = {\"width\": 10, \"height\": 10, \"N\": [5, 10, 20, 40, 80]}\n",
+    "\n",
+    "results_5s = mesa.batch_run(\n",
+    "    MoneyModel,\n",
+    "    parameters=params,\n",
+    "    iterations=100,\n",
+    "    max_steps=120,\n",
+    "    number_processes=1,\n",
+    "    data_collection_period=1,  # Important, otherwise the datacollector will only collect data of the last time step\n",
+    "    display_progress=True,\n",
+    ")\n",
+    "\n",
+    "results_5s_df = pd.DataFrame(results_5s)"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "execution_count": null
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [
+    "# Again filter the results to only contain the data of one agent (the Gini coefficient will be the same for the entire population at any time)\n",
+    "results_5s_df_filtered = results_5s_df[(results_5s_df.AgentID == 0)]\n",
+    "results_5s_df_filtered.head(3)"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "execution_count": null
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [
+    "# Create a lineplot with error bars\n",
+    "g = sns.lineplot(\n",
+    "    data=results_5s_df,\n",
+    "    x=\"Step\",\n",
+    "    y=\"Gini\",\n",
+    "    hue=\"N\",\n",
+    "    errorbar=(\"ci\", 95),\n",
+    "    palette=\"tab10\",\n",
+    ")\n",
+    "g.figure.set_size_inches(8, 4)\n",
+    "plot_title = \"Gini coefficient for different population sizes\\n(mean over 100 runs, with 95% confidence interval)\"\n",
+    "g.set(title=plot_title, ylabel=\"Gini coefficient\");"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In this case it looks like the Gini coefficient increases slower for smaller populations. This can be because of different things, either because the Gini coefficient is a measure of inequality and the smaller the population, the more likely it is that the agents are all in the same wealth class, or because there are less interactions between agents in smaller populations, which means that the wealth of an agent is less likely to change."
```

Comment:
The Gini coefficient being the way it is, is due to [preferential attachment](https://en.wikipedia.org/wiki/Preferential_attachment). The inequality scales faster than linearly against the number of nodes. I don't think this paragraph has incorporated preferential attachment in the explanation.

Reply:
Did you mean to reply to https://github.com/projectmesa/mesa/pull/1955#discussion_r1448832380 ? If you say so, you have to explicitly say it's a different way to visualize the same data.